kubernetes/template/DeploymentTemplates/clusterDefinition.json

```diff
@@ -6,7 +6,6 @@
             "orchestratorType": "Kubernetes",
             "orchestratorRelease": "",
             "kubernetesConfig": {
-                "kubernetesImageBase": "",

```

Comment:
we should keep this line if previous versions of the gallery item rely on this file